BibBuddy/app/src/main/res/values/strings.xml

```diff
@@ -137,16 +137,26 @@
     <string name="format_options">Format Options</string>
     <string name="format_arrow">Format Arrow</string>
 
-
     <!-- Export Dialog Alert Window Descriptions -->
     <string name="storage_permission_needed">Zugriff erforderlich</string>
     <string name="storage_permission_alert_msg">
         Zugriff auf den Gerätespeicher erforderlich,
```

Comment:
I think this is a bit long. I would either go with the first two lines ("Zugriff auf den Gerätespeicher erforderlich, um Dateien aus dieser App zu exportieren") or with the last two ("Erlaube den Zugriff und versuche nochmal, die Dateien zu exportieren"), but not all four ones, they kind of repeat themselves :)

Reply:
But if the user allows the permissions after the alert dialog is shown, then the user should select the "Export"-function of the menu again, since runtime permissions are not possible with "relationale permissions".

Reply:
I understand that, but four lines for a rather short toast are too long to be read I think.

Reply:
This is not used in toast, but in an alert dialog, that is shown in the middle of the screen when the user denies the permission that provides additional information on why the permission is needed and what the user should do after he/she allows the permission.

Reply:
- closed, we discussed this personally. Solution: Newly formulated and shorter text, see next commit :)